android-sdk/src/test/java/org/sagebionetworks/bridge/sdk/android/ExampleUnitTest.java

```diff
@@ -10,6 +19,9 @@
 public class ExampleUnitTest {
   @Test
   public void addition_isCorrect() throws Exception {
+
+    Logger log = LoggerFactory.getLogger(ExampleUnitTest.class);
```

Comment:
Does this mean Log statements using LoggerFactory work when called from unit tests now?